README.md

```diff
@@ -1,10 +1,19 @@
-# mumble-pahelper
+<p align="center"><img src = "resources/MumblePAHelper.png"/></p>
 
-Mumble Positional Audio Helper is a small tool for testing the functionality
-of positional audio plugins outside of Mumble.
+Mumble Positional Audio Helper is a small tool for testing the functionality of positional audio plugins outside of Mumble.
 
-You can use this tool to quickly see if your plugin is working as it should.
-Simply place it next to your plugin (or one level up from a /plugins folder
-and launch it.
+You can use this tool to quickly see if your plugin is working as it should.<br>
+Simply place it next to your plugin and launch it.
 
-![Screenshot of pahelper](https://github.com/mumble-voip/mumble-pahelper/raw/master/media/MumblePAHelper.png)
\ No newline at end of file
+## Default paths
```

Comment:
What about adding a note "You can find existing plugins in these folders" to indicate this is not where he has to place the PAHelper/his plugins to test?

Or did you mean to give these as a hint where to place them *after* testing with PAHelper?